examples/secretsmanager/v1beta1/secret-withreplica.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: secretsmanager.aws.upbound.io/v1beta1
+kind: Secret
+metadata:
+  name: example
```

nit: It would be better if we randomize the `metadata.name` field and add `example-id` annotation.

```suggestion
  name: example-${Rand.RFC1123Subdomain}
  annotations:
    meta.upbound.io/example-id: secretsmanager/v1beta1/secret
```